src/backend/distributed/commands/role.c

```diff
@@ -410,7 +412,7 @@ MakeVariableSetStmt(const char *config)
 	VariableSetStmt *variableSetStmt = makeNode(VariableSetStmt);
 	variableSetStmt->kind = VAR_SET_VALUE;
 	variableSetStmt->name = name;
-	variableSetStmt->args = list_make1(MakeSetStatementArgument(name, value));
+	variableSetStmt->args = MakeSetStatementArgument(name, value);
```

Comment:
would it make sense to rename this to its plural form? Especially since it is assigned to `args` which is plural on its own.
```suggestion
	variableSetStmt->args = MakeSetStatementArguments(name, value);
```